CHANGELOG.md

```diff
@@ -22,7 +22,14 @@ Classify the change according to the following categories:
     ### Fixed
     ### Deprecated
     ### Removed
-
+    
+## Develop - 2023-01-31
+ ##### Changed
+ - Updated the following default values:
+   - PV, Wind, Storage, CHP, GHP, Hot Water Storage, Cold Water Storage, Electric Storage: **federal_itc_fraction(PV,Wind, CHP,GHP)** and **total_itc_fraction(Hot Water Storage, Cold Water Storage, Electric Storage)** to 0.3 (30%)
+   - PV, Wind, Storage, CHP, GHP, Hot Water Storage, Cold Water Storage, Electric Storage: ***macrs_bonus_fraction** to 0.8 (80%)
+   - Wind ITC no longer determined based on size class
+   - Hot Water Storage and Cold Water Storage: **macrs_itc_reduction** to 0.5 (50%)
```

Comment:
@bpulluta could you add the update the Hot and Cold Water Storage macrs_option_years here as well?

Reply:
@adfarth addressed in 017201c953531f6ec824d5b726ea72a69dc11b94